### PR TITLE
fix(core): output git lfs error messages when there is an error

### LINKED ISSUE #1814: Errors not reported with renku storage commands
**Describe the bug**
If there's an issue when executing commands like `git lfs pull` the corresponding `renku storage pull` command does not output the stack trace. The renku command exits (successfully) instead of reporting the error.

**Additional context**
This happened in a couple of deployments when testing LFS and it was not working properly.

### PR DESCRIPTION
**Pull request by Panaetius**
closes #1814

### CODE REVIEW

File: renku/core/management/storage.py [hunk 1]
```diff
@@ -182,23 +189,37 @@ def track_paths_in_storage(self, *paths):
...
+            result = run(
+                self._CMD_STORAGE_UNTRACK + list(paths),
+                stdout=PIPE,
+                stderr=STDOUT,
+                cwd=self.path,
+                universal_newlines=True,
             )
+
+            if result.returncode != 0:
+                raise errors.GitLFSError("fError executing 'git lfs untrack: \n {result.stdout}")
```
Comment (mohammad-sdsc) [inline renku/core/management/storage.py:220]: "fError ... -> f"Error ...

**Review by mohammad-sdsc — COMMENTED** [review]
Look good! Just a minor typo.

File: renku/core/management/storage.py (hunk 1 above)
Comment (Panaetius) [reply renku/core/management/storage.py]: Good catch! Thank you

**Review by mohammad-sdsc — APPROVED** [review]
Thank you!

**Pull request merged**